package-lock.json

```diff
@@ -8283,14 +8283,12 @@
             "balanced-match": {
               "version": "1.0.0",
               "bundled": true,
-              "dev": true,
-              "optional": true
+              "dev": true
```

Comment:
I suspect this is an error. Run `npm install` again to see if they return.

Reply:
...bettER?

Reply:
Dunno. I'm sure it's fine.